modules/features/operations/value.go

```diff
@@ -437,3 +439,45 @@ func init() {
 }
 
 ////////////////////////////////////////////////////////////////////////////////
+
+type set struct {
+	flows.BaseFeature
+	vector map[interface{}]bool
+	set []interface{}
+}
+
+func (f *set) Start(context *flows.EventContext) {
+	f.BaseFeature.Start(context)
+	f.vector = make(map[interface{}]bool)
+	f.set = make([]interface{}, 0)
+}
+
+func (f *set) Stop(reason flows.FlowEndReason, context *flows.EventContext) {
+	f.SetValue(f.set, context, f)
+}
+
+func (f *set) Event(new interface{}, context *flows.EventContext, src interface{}) {
+	switch val := new.(type) {
+	case []byte:
+		if _, ok := f.vector[string(val)]; !ok {
```

Comment:
Use:
```go
if !f.vector[string(val)] {
```
instead. In case an element isn't found in a map, the types zero value is returned (`boolean` in this case returns `false`).

Reply:
Fixed